go/genesis/api/api.go

```diff
@@ -50,6 +58,19 @@ func (d *Document) UnmarshalCBOR(data []byte) error {
 	return cbor.Unmarshal(data, d)
 }
 
+// WriteFileJSON writes the genesis document into a JSON file.
+func (d *Document) WriteFileJSON(filename string) error {
+	docJSON, err := json.Marshal(d)
+	if err != nil {
+		return err
+	}
+
+	if err = ioutil.WriteFile(filename, docJSON, filePerm); err != nil {
+		return errors.Wrap(err, "WriteFileJSON: failed to write genesis file")
```

Comment:
For new code no need to use the external errors package (see #2057):
```suggestion
		return fmt.Errorf("WriteFileJSON: failed to write genesis file: %w", err)
```